server/validators/isValidExclusionZoneFile.ts

```diff
@@ -0,0 +1,30 @@
+import { registerDecorator, ValidationArguments, ValidationOptions } from 'class-validator'
```

Comment:
For the other validators we have a separate test file which unit tests them in isolation

Reply:
Thanks, I saw that late on, and I've added this requirement to the next ticket, CVSL-304.